plugins/connection/aws_ssm.py

```diff
@@ -662,9 +701,33 @@ def exec_command(self, cmd, in_data=None, sudoable=True):
                     break
                 stdout = stdout + line
 
-        stderr = self._flush_stderr(session)
+        # see https://github.com/pylint-dev/pylint/issues/8909)
+        return (returncode, stdout, self._flush_stderr(self._session))  # pylint: disable=unreachable
```

Comment:
should returncode have a properly defined default value?

Reply:
I am not sure if this is resolved. Am I missing something?

Reply:
@GomathiselviS the default value has been set to `None`, this is basically the case the process did not reached the end due to a timeout